app/actions/contact_identify_action.rb

```diff
@@ -5,6 +5,7 @@ def perform
     ActiveRecord::Base.transaction do
       @contact = merge_contact(existing_identified_contact, @contact) if merge_contacts?(existing_identified_contact, @contact)
       @contact = merge_contact(existing_email_contact, @contact) if merge_contacts?(existing_email_contact, @contact)
+      @contact = merge_contact(existing_phone_number_contact, @contact) if merge_contacts?(existing_phone_number_contact, @contact)
       update_contact
     end
```

Comment:
To get around with the code climate issue let's probably do something similar to this. define these private methods and move the existing logic into that

```suggestion
    ActiveRecord::Base.transaction do
      merge_if_existing_identified_contact
      merge_if_existing_email_contact
      merge_if_existing_phone_number_contact
      update_contact
    end
```